src/Microsoft.PowerShell.ConsoleHost/host/msh/StartTranscriptCmdlet.cs

```diff
@@ -137,6 +137,15 @@ public SwitchParameter IncludeInvocationHeader
             get; set;
         }
 
+        /// <summary>
+        /// Gets or sets whether to exclude transcript header.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter MinimalTranscriptHeader
```

Comment:
I suggest to use `-UseMinimalHeader`

Reply:
What is best practice? Why do we use a verb on first place? Can we use more short name `MinimalHeader`, `ShortHeader`, or `BriefHeader`?

Reply:
Readability (make the command use more descriptive) and preceding examples? There are many cmdlets with `-Use*` parameters.

Reply:
I'd give priority to IntelliSense (unique first char).

Reply:
It's not like you are adding the same prefix to multiple parameters of the same command. You will still get the unique first char when completing parameter for a specific command.